coderd/inboxnotifications.go

```diff
@@ -28,16 +29,63 @@ const (
 	notificationFormatPlaintext = "plaintext"
 )
 
+const (
+	FallbackIconWorkspace = "DEFAULT_WORKSPACE_ICON"
+	FallbackIconAccount   = "DEFAULT_ACCOUNT_ICON"
+	FallbackIconTemplate  = "DEFAULT_TEMPLATE_ICON"
+	FallbackIconOther     = "DEFAULT_OTHER_ICON"
```

Comment:
If these are values processed by "site", you should place them in codersdk as Enum, so they will be generated to TS.